tests/test.xpi.js

```diff
@@ -0,0 +1,238 @@
+import Xpi from 'xpi';
+import { Readable } from 'stream';
+
+
+const defaultData = {
+  compressionMethod: 8,
+};
+
+const chromeManifestEntry = Object.assign({}, defaultData, {
+  compressedSize: 138,
+  uncompressedSize: 275,
+  fileName: 'chrome.manifest',
+});
+
+const installRdfEntry = Object.assign({}, defaultData, {
+  compressedSize: 416,
+  uncompressedSize: 851,
+  fileName: 'install.rdf',
+});
+
+const chromeContentDir = {
+ compressionMethod: 0,
```

Comment:
I'm _guessing_ this means "no compression"? But again: not sure, feels like a magic constant.